### PR TITLE
full article

### PR DESCRIPTION
**Pull request by joshcolts18**

### CODE REVIEW

File: src/pages/blog/how-we-design-at-mintlify/index.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import { ArticleLayout } from '@/components/blog/ArticleLayout'
+
+export const meta = {
+  title: 'How We Approach Design At Mintlify',
+  description:
+    ': Good design is more than a process, it’s a headspace. Here are the building blocks of our approach to design. ',
```
Comment (handotdev) [inline src/pages/blog/how-we-design-at-mintlify/index.mdx:6]:
```suggestion
    ': Good design is more than a process. Here are the building blocks of our approach to design. ',
```

Comment (handotdev) [reply src/pages/blog/how-we-design-at-mintlify/index.mdx:6]: The headspace thing rubs off a bit cheesy

File: src/pages/blog/how-we-design-at-mintlify/index.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+<img
+  src="/static/images/blog/october-2023/desktop.png"
+  alt="blog thumbnail"
+  className="w-full rounded-md"
+/>
+
+When we ask our users why they moved to Mintlify, it’s often because of the user experience of our documentation. It’s something that we are always thinking about, and that we devote countless hours and iterations to.
+
+So what exactly makes good design? When you really boil it down, good design is all about taste.
```
Comment (handotdev) [inline src/pages/blog/how-we-design-at-mintlify/index.mdx:27]: Maybe add a GIF somewhere after this line or whenever relevant? Something that comes up about "taste" or "tasty" here maybe?

**Review by handotdev — APPROVED** [review]

File: src/pages/blog/how-we-design-at-mintlify/index.mdx (hunk 2 above)
Comment (joshcolts18) [reply src/pages/blog/how-we-design-at-mintlify/index.mdx:27]: Added GIF

**Pull request merged**